.buildkite/scripts/dra.sh

```diff
@@ -0,0 +1,64 @@
+set -uo pipefail
+
+DRY_RUN=""
+BRANCH="${BUILDKITE_BRANCH}"
+
+if [[ "${BUILDKITE_PULL_REQUEST:="false"}" != "false" || "${BUILDKITE_BRANCH}" == "ci_packaging_pipieline" ]]; then
+    BRANCH=main
+    DRY_RUN="--dry-run"
+    echo "+++ Running in PR and setting branch main and --dry-run"
+fi
+
+BEAT_VERSION=$(make get-version)
+
+CI_DRA_ROLE_PATH="kv/ci-shared/release/dra-role"
+
+# TODO use common function
+retry() {
```

Comment:
you can remove all this function once you've switched `retry 5` to `retry -t 5 --` as shown in the ticket below